routes/bots.js

```diff
@@ -132,18 +136,25 @@ botsRouter.route('/').post((req, res) => {
 /**
  * Adds an existing item to an existing bot
  */
-botsRouter.route('/addItem').post((req, res) => {
+botsRouter.route('/addItem').post(async (req, res) => {
 	const botId = req.body.botId;
 	const itemId = req.body.itemId;
 	const quantity = req.body.quantity;
 
 	if (!botId || !itemId || !quantity) {
-		res
+		return res
```

Comment:
Thank you for changing these, did you change all of them?

Reply:
Yes, I believe I changed all of them to include return.